<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PixmicroClass</class>
 <widget class="QMainWindow" name="PixmicroClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>527</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Pixmicro</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="VideoView" name="m_VideoView" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>48</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>176</red>
               <green>176</green>
               <blue>176</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>176</red>
               <green>176</green>
               <blue>176</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>176</red>
               <green>176</green>
               <blue>176</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>176</red>
               <green>176</green>
               <blue>176</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_Image"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="m_MenuCamera">
    <property name="title">
     <string>&amp;Camera</string>
    </property>
   </widget>
   <widget class="QMenu" name="m_MenuWindows">
    <property name="title">
     <string>&amp;Windows</string>
    </property>
   </widget>
   <widget class="QMenu" name="m_MenuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="m_MenuEdit"/>
   <addaction name="m_MenuCamera"/>
   <addaction name="m_MenuWindows"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="m_StatusBar"/>
  <action name="actionOpen_Image">
   <property name="text">
    <string>Open Image...</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="m_ActionWindowsCamera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="m_ActionWindowsEdgeDetect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edge detect</string>
   </property>
  </action>
  <action name="m_ActionWindowsFilters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filters</string>
   </property>
  </action>
  <action name="m_ActionWindowsSnapshot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Snapshot</string>
   </property>
  </action>
  <action name="m_ActionWindowsGallery">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gallery</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoView</class>
   <extends>QWidget</extends>
   <header>VideoView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Pixmicro.qrc"/>
 </resources>
 <connections/>
</ui>
